wordpress-mobile/gutenberg-mobile
# Update GB hash and fix Keyboard dismissed when enter.key is pressed

This PR updates the GB hash and fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/381 by reverting the changes introduced https://github.com/wordpress-mobile/gutenberg-mobile/pull/341

Testing steps:
- Tap on a para block
- Tap enter
- The new block should be created and the keyboard is up  

Repeat the step above for Heading and Title block.

GB side companion PR: https://github.com/WordPress/gutenberg/pull/12886

Make sure to re-open #308 when this is merged for the alpha.
